File: JAVASCRIPT/desafio1_inverterString.js
function main() {
  var string = "abc";
  console.log("STRING INICIAL: ", string);
  var array = transformarString(string);
  var arrayInvertido = inverter(array);
  var stringInvertida = tarnsformarArray(arrayInvertido);
  
  console.log("STRING FINAL: ", stringInvertida);
}

function tarnsformarArray(array) {
  return array.join("");
}

function transformarString(string) {
  return string.split("");
}

function inverter(array) {
  return array.reverse();
}

main();

File: PYTHON/desafio1_inverterString.py
def inverter(palavra):
    array = list(palavra)
    array.reverse()
    palavraInvertida = "".join(array)
    return palavraInvertida


def main():
    palavra = "abc"
    palavraInvertida = inverter(palavra)
    
    print("STRING INICIAL:" , palavra)
    print("STRING FINAL: ", palavraInvertida)


if __name__ == "__main__":
    main()